pkg/daemon/ceph/client/command.go

```diff
@@ -41,8 +42,7 @@ const (
 	Kubectl = "kubectl"
 	// CrushTool is the name of the CLI tool for 'crushtool'
 	CrushTool             = "crushtool"
-	CmdExecuteTimeout     = 1 * time.Minute
-	CephConnectionTimeout = "15" // in seconds
+	CephConnectionTimeout = 15 * time.Second
```

Comment:
How about defining the string version in one place so it avoids the conversion code everywhere else?
```suggestion
	CephConnectionTimeout = 15 * time.Second
	CephConnectionTimeoutSeconds = strconv.Itoa(int(CephConnectionTimeout.Seconds()))
```

Reply:
We can't use non-constant values like `Itoa`, so I'd like to keep `strconv` as is.

Reply:
It could be change to a `var`, but i'm ok if it stays like this.